homeassistant/components/group/entity.py

```diff
@@ -131,6 +131,7 @@ class Group(Entity):
     _unrecorded_attributes = frozenset({ATTR_ENTITY_ID, ATTR_ORDER, ATTR_AUTO})
 
     _attr_should_poll = False
+    active_domains: set[str]
```

Comment:
Please add a comment explaining what this attribute does

Reply:
done